src/mame/drivers/midvunit.cpp

```diff
@@ -35,10 +37,8 @@ Known to exist but not dumped:
 #include "includes/midvunit.h"
 #include "crusnusa.lh"
 
-
```

Comment:
Don't go stripping out the two-line gaps between logical sections of the file, it's unhelpful.

Reply:
I'm going to take note of that. Thanks.